x-pack/plugins/lens/public/react_embeddable/renderer/lens_custom_renderer_component.tsx

```diff
@@ -147,6 +148,7 @@ export function LensRenderer({
           ...initialStateRef.current,
           attributes: props.attributes,
         }),
+        hidePanelTitle: hidePanelTitles$,
```

Comment:
I don't love how we drop the `$` here on all the `BehaviorSubjects` but since it's already there for the other ones I can't complain.

I think it should be carried over anywhere we use an `Observable`/`BehaviorSubject`, i.e.

```ts
hidePanelTitle$: hidePanelTitles$,
```

Reply:
I know, if I'm not wrong these are interfaces from the PresentationPanel so we need to discuss that with @elastic/kibana-presentation

Reply:
Right, the naming conversion in the Presentation panels needs to be re-evaluated to ensure all observables contain `$` suffix. This is something that will be addressed during the clean-up phase of the embeddable refactor.